pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -3209,7 +3209,7 @@ func deleteRegistrationEntrySupport(tx *gorm.DB, entry RegisteredEntry) error {
 	return nil
 }
 
-func pruneRegistrationEntries(tx *gorm.DB, expiresBefore time.Time) error {
+func pruneRegistrationEntries(tx *gorm.DB, expiresBefore time.Time, ds *Plugin) error {
```

Comment:
Can we pass the logger here instead (e.g. `ds.log`)?

Reply:
Sure, thanks!